includes/ucf-section-posttype.php

```diff
@@ -21,6 +21,144 @@ public static function register() {
 				)
 			);
 			register_post_type( 'ucf_section', self::args( $labels ) );
+			add_action( 'add_meta_boxes', array( 'UCF_Section_PostType', 'register_metabox' ) );
+			add_action( 'save_post', array( 'UCF_Section_PostType', 'save_metabox' ) );
+		}
+
+		/**
+		 * Adds the UCF Section Assets metabox
+		 * @author Jim Barnes
+		 * @since 1.0.4
+		 **/
+		public static function register_metabox() {
+			add_meta_box(
+				'ucf_section_metabox',
+				'UCF Section Fields',
+				array( 'UCF_Section_PostType', 'register_fields' ),
+				'ucf_section',
+				'normal',
+				'low'
+			);
+		}
+
+		/**
+		 * The markup callback for the UCF Section Assets metabox
+		 * @author Jim Barnes
+		 * @since 1.0.4
+		 * @param $post WP_Post | The current post object
+		 * @return string | The function output is echoed
+		 **/
+		public static function register_fields( $post ) {
```

Comment:
Not really necessary for this PR, but at some point we should probably add some hook into the Page-Assets plugin to prevent it from also adding css/js fields to the section post type.